pkg/util/kubernetes/kubelet/kubelet.go

```diff
@@ -211,7 +211,7 @@ func (ku *KubeUtil) GetLocalPodList() ([]*Pod, error) {
 	// cache the podList to reduce pressure on the kubelet
 	cache.Cache.Set(podListCacheKey, pods, ku.podListCacheDuration)
 
-	return pods.Items, nil
+	return tmpSlice, nil
```

Comment:
we're doing `pods.Items = tmpSlice` before so I guess we cans stick with this line?